docker/generate_build_matrix.py

```diff
@@ -32,6 +32,14 @@ def _get_torchvision_version(pytorch_version: str):
     raise ValueError(f'Invalid pytorch_version: {pytorch_version}')
 
 
+def _get_pytorch_version(python_version: str):
```

Comment:
nit: since this can return multiple versions, should be `_get_pytorch_versions`?

Reply:
resolved